include/boost/parameter/are_tagged_arguments.hpp

```diff
@@ -43,6 +43,20 @@ namespace boost { namespace parameter {
     };
 }} // namespace boost::parameter
 
+#if defined(BOOST_PARAMETER_CAN_USE_MP11)
+#include <boost/mp11/integral.hpp>
+
+namespace boost { namespace parameter {
+
+    template <typename TaggedArg0, typename ...TaggedArgs>
+    using are_tagged_arguments_mp11 = ::boost::mp11::mp_bool<
```

Comment:
This shouldn't be needed. The original is usable as-is.

Reply:
I'll remove that and is_argument_pack_mp11 on the next pull request I submit.

Reply:
Okay, they're removed.